src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+from pathlib import Path
+
+import pytest
+
+from diffpy.labpdfproc.tools import set_output_directory
+
+params1 = [
+    ([None], [Path.cwd().resolve()]),
+    (["."], [Path.cwd().resolve()]),
+    (["new_dir"], [Path.cwd().resolve() / "new_dir"]),
+    (["new_dir.py"], [Path.cwd().resolve() / "new_dir.py"]),
```

Comment:
I think this example is confusing so I would just delete it.  It is also testing the same thing as the line above, unless a file exists with that name and then it is tested below.  

summary: delete